internal/compiler/widgets/cosmic-base/std-widgets-impl.slint

```diff
@@ -24,7 +24,6 @@ export global StyleMetrics  {
     out property <color> textedit-text-color: CosmicPalette.foreground;
     out property <color> textedit-background-disabled: CosmicPalette.control-disabled;
     out property <color> textedit-text-color-disabled: CosmicPalette.text-disabled;
-    out property <bool> dark-color-scheme: CosmicPalette.dark-color-scheme;
```

Comment:
This is - strictly speaking - a breaking change. StyleMetrics is undocumented, but accessible to apps. I'd rather keep things thing, but if you all feel like we should keep it, then I can keep this one here for compatibility.

Reply:
There is a lot of users out there:
https://github.com/search?q=StyleMetrics.dark-color-scheme+-repo%3Aslint-ui%2Fslint&type=code
Please keep it.

If anything we could deprecate it somehow.

Reply:
I've restored it and edited the description of https://github.com/slint-ui/slint/issues/3023